hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/metadata/SCMMetadataStoreImpl.java

```diff
@@ -192,7 +191,7 @@ public void start(OzoneConfiguration config)
 
       metaTable = META.getTable(store);
 
-      checkAndPopulateTable(moveTable, META.getName());
+      checkAndPopulateTable(metaTable, META.getName());
```

Comment:
Ouch...
Fortunately this affects just metrics as I see the tableMap that is populated here is not used elsewhere...

I believe this is irrelevant for the CLI tool, and this should be in a separate patch under a separate JIRA describing the problem we fix, and we should not hide it here.

Reply:
FYI, this is already fixed in HDDS-7490.